pkg/gui/branches_panel.go

```diff
@@ -124,6 +124,32 @@ func (gui *Gui) handleCreatePullRequestPress(g *gocui.Gui, v *gocui.View) error
 	return nil
 }
 
+func (gui *Gui) handleGitFetch(g *gocui.Gui, v *gocui.View) error {
+	if err := gui.createMessagePanel(g, v, "", gui.Tr.SLocalize("FetchWait")); err != nil {
+		return err
+	}
+	go func() {
+		unamePassOpend, err := gui.fetch(g, v, true)
+		if err != nil {
+			errMessage := err.Error()
+			if errMessage == "exit status 128" {
+				errMessage = gui.Tr.SLocalize("PassUnameWrong")
+			}
+			_ = gui.createErrorPanel(g, errMessage)
+		}
+		if unamePassOpend {
+			_, _ = g.SetViewOnBottom("pushPassUname")
```

Comment:
do we need to call SetViewOnBottom here when we're deleting the view anyway?

Reply:
I though it was needed otherwise it would break but this is totally unneeded after testing, I'll remove the `SetViewOnBottom`